chapters/introduction.tex

```diff
@@ -97,59 +77,76 @@ \section{Outline of this book}
 handling data processing and analytical tasks.
 
 
-\section{Adopting reproducible workflows}
+\section{Adopting reproducible tools}
 We will provide free, open-source, and platform-agnostic tools wherever possible,
```

Comment:
I think this discussion should come here:

We assume througout all of this book that you are going to do nearly all of your data work though code.
It may be possible to perform all relevant tasks through the user interface in some statistical software, 
or even through less field-specific software such as Excel.
However, we strongly advise against it.
The reason for that are the transparency, reproducibility and credibility principles discussed in chapter 1.
Writing code creates a record of every task you performed. 
It also prevents direct interaction with the data files that could lead to non-reproducible steps.
Think of the code as a recipe to create your results:
other people can follow it,
reproduce it,
and even disagree with your the amount of spices you added
(or some of your coding decisions).
For these reasons, code is now considered an essential component of a research output.

Reply:
And by "here" I mean the first paragraph of this subsection.

Reply:
This is the first step to solve #355

Reply:
50ca90d3cd1c2839d4bc5a0d9f26a110459bc812